### ISSUE TITLE
Veyon fails under Deepin Desktop

### ISSUE THREAD

**Issue by sir01g** [veyon/veyon#659]
I installed Veyon master on xubuntu 20.04 with Deepin Desktop installed too.
It worked very good with another computers with the same configuration when I logged in them to XFCE.
But if I logged in to Deepin Desktop, the Veyon Master said for that computer that the service is not running or not available. After it I logged out. At the lightdm greetings, the computer was available in the Master. But when I logged into XFCE again, the computer wasn't available only after restarting the Veyon-service manually.
Every time I logging into Deepin it destroys the connection and I have to restart the service to connect again to the Master. 
If I use onle XFCE on the machine, it works good.. until I logging into Deepin. In Deepin I can't make the computer the connect to the Master (I restarted the service many times, and it doesn't work). Even when I logging back to XFCE, the service must be restarted after a Deepin session to connect to the Master.
I have no idea where to find the problem.
The Master works under Deepin but the clients not...

**Comment by sir01g** [comment]
Update: I installed an UbuntuDDE client. It doesn't work either. Not even the login screen showed up in the Master: after installing Veyon-Service in the UbuntuDDE and rebooted it, the icon in Master turned gray from red, but nothing else.
I checked: I can view the client machine with connecting to the X11VNc server directly from a VNC viewer. So the problem isn't originated from VNC, it's Veyon problem.

**Comment by tobydox** [comment]
Can you please try to update to version 4.5.0 and report if this problem still persists? We have lot's of Linux and user-session-related fixes in 4.5.0.

**Comment by sir01g** [comment]
I've had only one attempt this far, but it wasn't success. On one computer (which worked under XFCE) I upgraded to the new version and tried on deepin. That computer computer doesn't work since than. Even when I downgraded to the original version. It doesn't work even on XFCE.
At home in a virtual environment I tried once successfully. But only once. The next try on the same virtual machine it never worked since. So I suspect the deepin breaks somehow the Veyon connection.
When the epidemic ends and I can use again the system at work I want to install a new Linux (maybe Mint) on one computer. Than I will try the new Veyon on this freshly installed machine with XFCE and Deepin too. If it works I will clone the system to the other computers. I will report the result when it it would be done.
